iphone/Classes/TiThreading.h

```diff
@@ -40,14 +40,14 @@
   }
 
 //TODO: Now that we have TiThreadPerform, we should optimize this out.
-#define ENSURE_UI_THREAD_WITH_OBJ(x, y, z)                                         \
-  if (![NSThread isMainThread]) {                                                  \
+#define ENSURE_UI_THREAD_WITH_OBJ(x, y, z)                                          \
```

Comment:
clang-format is complaining (and rightfully so) that the line continuations don't line up